src/parameters.toml

```diff
@@ -2139,6 +2151,18 @@ value = 0.12
 type = "float"
 description = "Coefficient for the b/w^2 term in the detrainment closure. See Tan et al. (2018) [https://doi.org/10.1002/2017MS001162], Eq 27."
 
+[max_area_limiter_scale]
+alias = "max_area_limiter_scale"
+value = 0.1
+type = "float"
+description = "Constant coefficient that scales the maximum area limiter term in detrainment. Parameter not described in the literature."
+
+[max_area_limiter_power]
+alias = "max_area_limiter_power"
+value = 10
+type = "float"
+description = "Constant coefficient for the exponent in the maximum area limiter term in detrainment. Parameter not described in the literature."
+
```

Comment:
I can also not add these two parameters and use the ones we have [here](https://github.com/CliMA/CLIMAParameters.jl/blob/945eb72ea84cf5ad8231bcdd7715e8a57dd11e45/src/parameters.toml#L2250-L2260). Just the description is not accurate. What do you think is better?

Reply:
Maybe we could:
1. Add new ones with the correct descriptions
2. Add the ones we need to delete to this garbage collecting issue: https://github.com/CliMA/CLIMAParameters.jl/issues/99
3. Promise to actually handle this issue before the next sprint ;) @nefrathenrici , @szy21  - What do you think?